backend-diagram-converter/core/src/main/resources/message-templates.properties

```diff
@@ -2,6 +2,7 @@
 # Building blocks
 #
 template.expression-transformation-result=Please review transformed expression: '{{ oldExpression }}' -> '{{ newExpression }}'.
+template.expression-transformation-hint=Transformed expression: '{{ oldExpression }}' -> '{{ newExpression }}'.
```

Comment:
Could we also stick to `template.expression-transformation-result` here?

Reply:
No, the messages should be different.

I think, `REVIEW: Please review transformed expression ...`

and `TASK: Transformed expression ... 'execution' is not available in FEEL` 

is important to distinguish.